group_vars/x86_64.yml

```diff
@@ -4,5 +4,6 @@ hostname: ideasbox.lan
 hotspot_name: ideasbox
 wpa_pass: idb258456
 ideascube_project_name: ideasbox
+ideascube_branding: ideasbox
```

Comment:
Heh, that simple. Now I feel bad for not thinking about it. :stuck_out_tongue: 

Thanks!